tasks/update_go.py

```diff
@@ -49,7 +49,7 @@ def update_go(
     ctx: Context,
     version: str,
     image_tag: str,
-    test_version: Optional[bool] = False,
```

Comment:
The default value of `test-version` in `buildimages.update` is `True`, this change aligns both invoke tasks so that their argument is the same.